src/scss/components/file/_hx-file-control.scss

```diff
@@ -0,0 +1,89 @@
+@import "components/form/config";
```

Comment:
Remove this import. 

* This will/should be imported in `src/scss/components/file/_config.scss`

Reply:
Import statement removed.